.github/workflows/test.yml

```diff
@@ -0,0 +1,17 @@
+name: Test
+on: push
```

Comment:
You likely want this on `pull_request`, not `push`.

Reply:
Very yes, very yes I do, thank you 😄